backend/endpoints/jokes/blueprint.py

```diff
@@ -1,11 +1,71 @@
-from flask import Blueprint, make_response
+from flask import Blueprint, make_response, request
+from utils.eeljokes import EelJokes, db
+import json
 
 bp = Blueprint(
     "eelbot_joke",
     __name__,
     url_prefix="/joke",
 )
 
-@bp.route("/")
-def main():
-    return make_response(("Hello Eels!", 200))
\ No newline at end of file
+@bp.route("/", methods=['POST'])
+def create_joke():
+    json_data = request.json
+    new_task = EelJokes(JokeText=json_data[0], JokeTextLine2=json_data[1])
+
+    try:
+        db.session.add(new_task)
```

Comment:
```suggestion
    new_joke = EelJokes(JokeText=json_data[0], JokeTextLine2=json_data[1])

    try:
        db.session.add(new_joke)
```

Name to something more appropriate, you are creating a new object and adding it to the collection in the ORM not queuing up a set of tasks to execute (at least not directly).